### DISCUSSION TITLE
Connect to Azure PostgresDB from App Service without secrets using a managed identity

### DISCUSSION THREAD

**Discussion by jeelmakani** [prisma/prisma#20267] [Q&A]
currently, using prisma for my project and i'm using Azure Active directory Authentication for Database.
Here is the link of azure docs.
https://learn.microsoft.com/en-us/azure/postgresql/single-server/concepts-azure-ad-authentication
also here is link for azure tutorial for managed identity service.
https://learn.microsoft.com/en-us/azure/app-service/tutorial-connect-msi-azure-database?tabs=sqldatabase%2Csystemassigned%2Cnetfx%2Cwindowsclient

since, i am using managed identity service, encountering issue with dynamic DATABASE_URL. because password for DB is accessToken, which expires in couple of minutes. 
Using Nestjs as backend.

Can some help me with this, how to make it work?

**Comment by nurul3101** [comment]
Hello @jeelmakani 👋 

Azure Active Directory Authorization isn't supported natively by Prisma yet.

We have a feature request for it:

- https://github.com/prisma/prisma/issues/12562

If you could add a 👍 to the feature request, then it would help our product team in prioritising it.

Hope this helps! Please let us know in case you have any queries, if not, please consider marking the discussion as answered/closed.
